src/components/MainNavigation/README.mdx

```diff
@@ -1,35 +1,21 @@
-The Main Navigation is a Horizontal Menu which consists of multiple
-clickable items placed at the top of the page. The navigation stays
-unchanged when browswing through the site and is present on every page.
-The currently selected item is usually highlighted.
-
-The MainNavigation is used across all applications, it is the starting point of the application and offers the top level navigation items.
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
 
 ### Usage
 
+The MainNavigation is used across all applications, it is the starting point of the application and offers the top level navigation items.
+
 import { List, ListItem } from '../List';
 
-<List kind="bullet">
-  <ListItem kind="checkmark">Internal and external applications</ListItem>
-  <ListItem kind="checkmark">
-    The MainNavigation can be used for both internal and external websites.
-  </ListItem>
-  <ListItem kind="checkmark">
-    The MainNavigation can be used for both internal and external websites.
-  </ListItem>
-</List>
-
-### Do not use
-
-<List kind="bullet">
-  <ListItem kind="cross">
-    External websites (wfp.org, etc.): The header might not be the right
-    solution for external websites. Please consult the #ux-ui channel on Slack.
-  </ListItem>
-  <ListItem kind="cross">
-    Do not use WFP's logo inside the MainNavigation.
-  </ListItem>
-</List>
+#### When to use
```

Comment:
Shall we use the regular list for `When to use`, `When not to use` here or the list with a `checkmark` or `cross` icons?

Reply:
sure